src/server/csp.ts

```diff
@@ -65,24 +65,24 @@ const assetHashes = (assets: string[]): string =>
 //     child-src https: blob:
 // `.trim()
 
-const buildCsp = ({ styles, scripts }: Assets, twitter: boolean): string => `
+const buildCsp = ({ styles, scripts }: Assets, twitter: boolean, instagram: boolean, youtube: boolean, spotify: boolean): string => `
```

Comment:
We should think if there is a better way to do this rather than passing loads of parameters into the function, @JamieB-gu any ideas?